### PR TITLE
Implement address cards

### PR DESCRIPTION
**Pull request by nop33**
https://user-images.githubusercontent.com/1579899/211887531-76f1cc3b-bbff-484e-b781-6f53f8009304.mp4

### CODE REVIEW

File: src/components/Button.tsx [hunk 1]
```diff
@@ -24,10 +24,9 @@ import styled, { css } from 'styled-components'
 import { sectionChildrenVariants } from './PageComponents/PageContainers'
 
 interface ButtonProps extends HTMLMotionProps<'button'> {
-  secondary?: boolean
-  alert?: boolean
+  mode?: 'primary' | 'secondary' | 'transparent'
```
Comment (nop33) [inline src/components/Button.tsx:27]: Any better idea for this prop name? `type` is taken by `HTMLMotionProps`...

Comment (mvaivre) [reply src/components/Button.tsx:27]: role?

**Comment by mvaivre** [comment]
Super nice evolution! 

Regarding the layout: I've been trying the UI with many addresses and it's a bit hard to read when the cards are spread over multiple rows. 
I initially intended the address list to be scrollable horizontally : this could be the opportunity to see how it would behave?
If the page has no overflow (should be the case here), then we can allow to scroll with the mouse wheel without pressing CTRL.

File: src/components/Button.tsx (hunk 1 above)
Comment (nop33) [reply src/components/Button.tsx:27]: done!

**Comment by nop33** [comment]
> If the page has no overflow (should be the case here)

FWIT, the behavior now is that you can scroll vertically:

https://user-images.githubusercontent.com/1579899/212046202-488672e1-6d58-42f2-81eb-536815dcf893.mp4

**Comment by mvaivre** [comment]
> FWIT, the behavior now is that you can scroll vertically:

Yes I know, but I find it hard to parse with many addresses.

image

Let's start tweaking the cards' design before trying horizontal scrolling.

**Review by mvaivre — APPROVED** [review]
👍 👏
